packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -454,7 +458,10 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
       direction: vtkImageData.getDirection(),
       scalarData: vtkImageData.getPointData().getScalars().getData(),
       imageData: actor.getMapper().getInputData(),
-      metadata: { Modality: this.modality },
+      metadata: {
+        Modality: this.modality,
+        FrameOfReferenceUID: this.getFrameOfReferenceUID(),
```

Comment:
Since you are tweaking this, do you need sop class uids - some modalities like MG really need that list for things like figuring out if it is a CT type series or is radiographic.

Reply:
That is not a breaking change, so when that need arise we can add that